app/services/register_and_partner_api/sync_users.rb

```diff
@@ -105,6 +107,24 @@ def self.assign_user_attributes(attributes, user, profile)
       profile.cohort = get_cohort(attributes)
     end
 
+    def self.create_or_destroy_cip_change_message(user, attributes)
```

Comment:
User CIPs can be nil - a quirk of how R&P enter data.

When we change from `nil` to something, we probably don't want to create a guidance message. When we change from something to `nil`, eh, I don't know, probably worth asking Isabelle.

Reply:
Thanks, didn't know about this 👍